src/js/profile.js

```diff
@@ -87,10 +87,24 @@ $(function() {
                 .fail(xhr => complain(xhr.responseText))
         })
 
-        $("#input0").autocomplete({
-            source: friends,
-            appendTo: "#modal"
+        $("#input0").attr("data-list", friends );
+
+        new Awesomplete( ("#input0"), {
+            
+            filter: function(text, input) {
+                return Awesomplete.FILTER_CONTAINS(text, input.match(/[^,]*$/)[0]);
```

Comment:
adding to collection/ users and will update the code now.
This short notice should be commented in the code file or on github? or both?